src/modules/map/components/footer/MapFooterAttributionList.vue

```diff
@@ -62,6 +63,7 @@ export default {
                                 name: attribution.name,
                                 url: attribution.url,
                                 hasDataDisclaimer: this.hasDataDisclaimer(layer.id),
+                                isExternalDataLocal: this.isLocalFile(layer),
```

Comment:
then it would also make sense to match the new name of the props in ThirdPartyDisclaimer and have the source property named the same